frontend/src/pages/modelRegistry/screens/EmptyRegisteredModels.tsx

```diff
@@ -0,0 +1,28 @@
+import {
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  EmptyStateVariant,
+  Title,
+} from '@patternfly/react-core';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import * as React from 'react';
+
+type EmptyRegisteredModelsType = {
+  preferredModelRegistry?: string;
+};
+const EmptyRegisteredModels: React.FC<EmptyRegisteredModelsType> = ({ preferredModelRegistry }) => (
+  <EmptyState variant={EmptyStateVariant.full} data-testid="no-registered-models">
+    <EmptyStateIcon icon={PlusCircleIcon} />
+    <Title headingLevel="h5" size="lg">
+      No models in model registry
```

Comment:
```suggestion
      No models in selected registry
```
According to the [UX mock](https://www.figma.com/proto/KTsYhANYQqB0NUrXW7wQH1/Model-Registry?page-id=568%3A44455&type=design&node-id=5015-224143&viewport=2345%2C828%2C0.24&t=7VDnlaMsA8Cun4vu-1&scaling=min-zoom&starting-point-node-id=5015%3A224117&show-proto-sidebar=1)